spring-boot-modules/spring-boot-groovy/pom.xml

```diff
@@ -71,8 +78,9 @@
 
     <properties>
         <start-class>com.baeldung.springwithgroovy.SpringBootGroovyApplication</start-class>
-        <groovy.version>3.0.13</groovy.version>
-        <gmavenplus-plugin.version>1.9.0</gmavenplus-plugin.version>
+        <groovy.version>4.0.11</groovy.version>
+        <gmavenplus-plugin.version>3.0.2</gmavenplus-plugin.version>
```

Comment:
Try removing explicit rest-assured version - it already comes transitively from spring-boot-dependencies of parent-boot-3

Reply:
sure.